{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"authenticationFailed"},{"kind":"text","text":"("},{"kind":"externalParam","text":"error"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"Error","preciseIdentifier":"s:s5ErrorP"},{"kind":"text","text":")"}],"languages":["swift"],"platforms":["iOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/zendesksdk\/zendeskevent\/authenticationfailed(error:)"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/com.zendesk.Zendesk\/documentation\/ZendeskSDK\/ZendeskEvent\/authenticationFailed(error:)","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"Invoked when a rest call fails for authentication reasons."}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"authenticationFailed"},{"kind":"text","text":"("},{"kind":"externalParam","text":"error"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"Error","preciseIdentifier":"s:s5ErrorP"},{"kind":"text","text":")"}],"title":"ZendeskEvent.authenticationFailed(error:)","roleHeading":"Case","role":"symbol","symbolKind":"case","externalID":"s:10ZendeskSDK0A5EventO20authenticationFailedyACs5Error_p_tcACmF","modules":[{"name":"ZendeskSDK"}]},"hierarchy":{"paths":[["doc:\/\/com.zendesk.Zendesk\/documentation\/ZendeskSDK","doc:\/\/com.zendesk.Zendesk\/documentation\/ZendeskSDK\/ZendeskEvent"]]},"references":{"doc://com.zendesk.Zendesk/documentation/ZendeskSDK":{"role":"collection","title":"ZendeskSDK","abstract":[],"identifier":"doc:\/\/com.zendesk.Zendesk\/documentation\/ZendeskSDK","kind":"symbol","type":"topic","url":"\/documentation\/zendesksdk"},"doc://com.zendesk.Zendesk/documentation/ZendeskSDK/ZendeskEvent":{"role":"symbol","title":"ZendeskEvent","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"ZendeskEvent"}],"abstract":[{"type":"text","text":"The set of events that can be emitted from Zendesk SDK."}],"identifier":"doc:\/\/com.zendesk.Zendesk\/documentation\/ZendeskSDK\/ZendeskEvent","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"ZendeskEvent"}],"url":"\/documentation\/zendesksdk\/zendeskevent"},"doc://com.zendesk.Zendesk/documentation/ZendeskSDK/ZendeskEvent/authenticationFailed(error:)":{"role":"symbol","title":"ZendeskEvent.authenticationFailed(error:)","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"authenticationFailed"},{"kind":"text","text":"("},{"kind":"externalParam","text":"error"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"Error","preciseIdentifier":"s:s5ErrorP"},{"kind":"text","text":")"}],"abstract":[{"type":"text","text":"Invoked when a rest call fails for authentication reasons."}],"identifier":"doc:\/\/com.zendesk.Zendesk\/documentation\/ZendeskSDK\/ZendeskEvent\/authenticationFailed(error:)","kind":"symbol","type":"topic","url":"\/documentation\/zendesksdk\/zendeskevent\/authenticationfailed(error:)"}}}